src/chrome/content/rules/Maybank2u.com.ph.xml

```diff
@@ -5,15 +5,17 @@
 
 	<target host="maybank2u.com.ph" />
 	<target host="www.maybank2u.com.ph" />
-
 		<test url="http://maybank2u.com.ph/resources/my/img/common/logo-maybank_m.png" />
 		<test url="http://www.maybank2u.com.ph/resources/my/img/common/logo-maybank2u_m.png" />
 
-	<!-- 404 on main page -->
-	<exclusion pattern="^http://(www\.)?maybank2u\.com\.ph/$" />
-
 	<securecookie host=".+" name=".+" />
 
+	<!-- only RIBNOW, resources and ph_maybank_gif directories are functional -->
+	<exclusion pattern="^http://(www\.)?maybank2u\.com\.ph/(?!RIBNOW|resources|ph_maybank_gif)" />
```

Comment:
The negative lookahead operator should be [avoided](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#regex-conventions) in favor of a more restrictive redirect rule.

Reply:
And a trailing slash is missing.